### ISSUE TITLE
Change styling of Navbar

### ISSUE THREAD

**Issue by Agrimaagrawal** [Balastrong/github-stats#48]
Words written on navbar are not visible I can change its styling

**Comment by Balastrong** [comment]
Hey @Agrimaagrawal, welcome!

That's right, styles aren't properly supported yet, I think dark theme should be ok-ish but light theme might be broken.
Would you like to work on this?

**Comment by Agrimaagrawal** [comment]
Thanks @Balastrong 
Yes I want to work on this

**Comment by Balastrong** [comment]
Fantastic! I'll assign you the issue :)

Do you already have a plan on how to handle this?

**Comment by Agrimaagrawal** [comment]
No, I don't have any plan yet but I will try to handle it as soon as I can

**Comment by Balastrong** [comment]
Ok, take your time :)
